code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -193,6 +193,39 @@
 		else
 			icon_state = "ramenempty"
 
+/obj/item/reagent_containers/food/drinks/energy
+	name = "Energy Drink"
+	desc = "A heart attack that fits in your pocket."
+	icon_state = "energy_drink"
+	center_of_mass = list("x"=15, "y"=13)
+	preloaded_reagents = list("sugar" = 10, "nuka_cola" = 20)
+	spawn_tags = SPAWN_TAG_JUNKFOOD
+	rarity_value = 15
```

Comment:
*"how often an item is spawned (based on its rarity) is inversely proportional to its rarity value"*

Reply:
will 40 suffice?

Reply:
edrink is supposed to be rare, kind of.